### PR TITLE
Add MetricSnapshotDataFrame,MetricRangeDataFrame basic usage to README

### LINKED ISSUE #157: Add `MetricSnapshotDataframe` basic usage docs to README
Similar to the `Metric` class usage we need basic usage instructions for the `MetricSnapshotDataframe` class.

### PR DESCRIPTION
**Pull request by chauhankaranraj**
Closes #157 
Adds a small description and example usage of the two modules.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,6 +142,58 @@ my_metric_object = metrics_object_list[1] # one of the metrics from the list
...
+)
+metric_df = MetricRangeDataFrame(metric_data)
+metric_df.head()
+""" Output:
++------------+------------+-----------------+--------------------+-------+
+|            |  __name__  | cluster         | label_2            | value |
++-------------------------+-----------------+--------------------+-------+
+| timestamp  |            |                 |                    |       |
++============+============+=================+====================+=======+
+| 1577836800 |   __up__   | cluster_id_0    | label_2_value_2    | 0     |
```
Comment (4n4nd) [inline README.md:189]: i think the metric name should be `up` instead of `__up__`

**Review by 4n4nd — CHANGES_REQUESTED** [review]
one minor change requested, other than that looks good

File: README.md (hunk 1 above)
Comment (chauhankaranraj) [reply README.md:189]: Yeah, looks like a typo. Lemme fix that.

**Review by 4n4nd — APPROVED** [review]
lgtm :+1:

**Pull request merged**